src/model.py

```diff
@@ -1785,6 +1784,10 @@ def forward(self, batch):
 
         input_ids = batch[0].to(torch.device(self.device))
         attention_mask = batch[1].to(torch.device(self.device))
+        true_labels = {
+            "expression": batch[2].to(torch.device(self.device)),
+            "target": batch[5].to(torch.device(self.device))
+        }
```

Comment:
Really only need expression, but added target for easier testing other transmission types